src/main/java/eggventory/parsers/Parser.java

```diff
@@ -124,6 +124,8 @@ private Command handleListInput(String listInput) throws BadInputException,
     public Command parse(String userInput) throws Exception {
         Command userCommand;
 
+        System.out.println("got input:" + userInput);
```

Comment:
should parser be calling system.out.print directly?

Reply:
oh my bad, this was a debug statement. I'll remove in my next PR